src/lib/LUA/lua.cpp

```diff
@@ -188,13 +189,20 @@ uint8_t getLuaWarningFlags(void){ //return an unsppressed warning flag
   return luaWarningFlags & suppressedLuaWarningFlags;
 }
 
+static void updateElrsFlags()
+{
+  setLuaWarningFlag(LUA_FLAG_MODEL_MATCH, connectionState == connected && connectionHasModelMatch == false);
+  setLuaWarningFlag(LUA_FLAG_CONNECTED, connectionState == connected);
+  setLuaWarningFlag(LUA_FLAG_ISARMED, IsArmed());
+}
+
 void sendELRSstatus()
 {
   constexpr const char *messages[] = { //higher order = higher priority
     "",                   //status2 = connected status
     "",                   //status1, reserved for future use
     "Model Mismatch",     //warning3, model mismatch
-    "[ ! Armed ! ]",           //warning2, reserved for future use
+    "[ ! Armed ! ]",      //warning2, reserved for future use
```

Comment:
It's no longer "reserved for future use" 😄

Reply:
😆 you're right!